vault/resource_jwt_auth_backend.go

```diff
@@ -110,6 +110,14 @@ func jwtAuthBackendResource() *schema.Resource {
 				Description: "The CA certificate or chain of certificates, in PEM format, to use to validate connections to the JWKS URL. If not set, system certificates are used.",
 			},
 
+			"jwks_pairs": {
```

Comment:
When did we add this field? Was it in recent Vault versions, or? Wondering if we should gate it with a version check or not.

Reply:
Good call, officially TFVP v5 supports Vault versions >= 1.15.x, so we can assume that customers should at least be on that minimum Vault version when using this feature. Doing a little digging in the docs, looks like this was added in Vault 1.16, so a version check might be a good addition!